examples/webpack.config.js

```diff
@@ -2,14 +2,14 @@
 
 const webpack = require('webpack');
 const path  = require('path');
-const Extract = require('extract-text-webpack-plugin');
```

Comment:
The old plugin is deprecated, and replaced by mini-css-extract-plugin.

Reply:
Actually the new plugin doesn't seem to work, I'm on it.

Reply:
Actually this works when I use NODE_ENV=production, this is a change from what's on master where the plugin was running in development too. I believe it's more correct this way.

With NODE_ENV=production all paths are prefixed with /react-contextmenu so it's not easy to test locally, but it should work once deployed to a github pages with a repository with this name. We could try that later.

Reply:
I see, thanks for letting me know.